packages/e2e-tests/specs/editor/various/post-editor-template-mode.test.js

```diff
@@ -1,377 +0,0 @@
-		await activateTheme( 'twentytwentyone' );
-		await createNewPost();
-		// Create a random post.
-		await page.type( '.editor-post-title__input', 'Another FSE Post' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'Hello World' );
-
-		// Unselect the blocks.
-		await page.evaluate( () => {
-			wp.data.dispatch( 'core/block-editor' ).clearSelectedBlock();
-		} );
-
-		// Save the post
-		// Saving shouldn't be necessary but unfortunately,
-		// there's a template resolution bug forcing us to do so.
-		await saveDraft();
-		await page.reload();
-
-		await createNewTemplate( 'Blank Template' );
-
-		// Edit the template.
-		await insertBlock( 'Paragraph' );
-		await page.keyboard.type(
-			'Just a random paragraph added to the template'
-		);
-
-		// Save changes.
-		const publishButton = await page.waitForXPath(
-			`//button[contains(text(), 'Publish')]`
-		);
-		await publishButton.click();
-		const saveButton = await page.waitForXPath(
-			`//div[contains(@class, "entities-saved-states__panel-header")]/button[contains(text(), 'Save')]`
-		);
-		await saveButton.click();
-		// Avoid publishing the post
-		const cancelButton = await page.waitForXPath(
-			`//button[contains(text(), 'Cancel')]`
-		);
-		await cancelButton.click();
-
-		// Preview changes.
-		const previewPage = await openPreviewPage();
-		await previewPage.waitForSelector( '.wp-site-blocks' );
-		const content = await previewPage.evaluate(
-			() => document.querySelector( '.wp-site-blocks' ).innerHTML
-		);
-
-		expect( content ).toMatchSnapshot();
-	} );
-} );
-
-describe( 'Delete Post Template Confirmation Dialog', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'gutenberg-test-block-templates' );
-		await deleteAllTemplates( 'wp_template' );
-		await deleteAllTemplates( 'wp_template_part' );
-		await activateTheme( 'twentytwentyone' );
-		await createNewPost();
-		// Create a random post.
-		await page.type( '.editor-post-title__input', 'Just an FSE Post' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'Hello World' );
-
-		// Save the post
-		// Saving shouldn't be necessary but unfortunately,
-		// there's a template resolution bug forcing us to do so.
-		await saveDraft();
-		await page.reload();
-		// Unselect the blocks.
-		await page.evaluate( () => {
-			wp.data.dispatch( 'core/block-editor' ).clearSelectedBlock();
-		} );
-	} );
-
-	afterAll( async () => {
-		await activateTheme( 'twentytwentyone' );
-		await deactivatePlugin( 'gutenberg-test-block-templates' );
-	} );
-
-	[ 'large', 'small' ].forEach( ( viewport ) => {
-		it( `should retain template if deletion is canceled when the viewport is ${ viewport }`, async () => {
-			await setBrowserViewport( viewport );
-
-			const isWelcomeGuideActive = await page.evaluate(
-				() =>
-					!! wp.data
-						.select( 'core/preferences' )
-						.get( 'core/edit-post', 'welcomeGuide' )
```

Comment:
But here we're accessing the feature flag from the `core/preferences` store. (2/2)